### ISSUE TITLE
[BUG] Layer fails to load through GeoCore id config and using URL [OSDP-1801]  (24 layers)

### ISSUE THREAD

**Issue by MLAuger** [Canadian-Geospatial-Platform/geoview#2604]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Failed to load through config using GeoCore id and using URL.  Gets the "failed to load on map mapWM" error message.

Might be CORS-related issue.

Concerned UUIDs:

de50ac30-ab0d-4bf2-b9af-362977edde3a
3862c9fa-dbeb-4f00-ac03-c5da6551bf00
71c25df1-0577-43f7-b9f8-95cc321e7cbc
79dcbe44-4a86-4ac0-8eec-135f80a9b843
eaf92ab8-c744-4f8f-a280-6987d4f6005e
8abcbe96-44e7-406e-8e05-54b32a3554bc
94e38a40-085a-4c8a-a7c4-5b46e03702c3
6b1408b4-ea09-4a6a-b5dd-3db3493e1218
8ce8abe1-d644-4134-baa9-389eebd63a05
522b07b9-78e2-4819-b736-ad9208eb1067
5a316fdc-3237-4ace-831e-67b4ca26a248
9eb51c7b-5336-4b50-a9f8-a6791bad15a9
6f7d1a39-fea8-4578-b1d0-766d7e92ac7a
2ebfa295-c51a-467c-8c38-823f27ed2a15
9e1507cd-f25c-4c64-995b-6563bf9d65bd
01e01816-9f08-49eb-a5ac-7480195d90d4
41d008f7-f083-4801-b7c0-c2bf5a83a757
e4d63d01-1e06-bb7b-1b49-ca6a88bf771b
77f5bbd2-4f24-466f-b309-5b73659d32a3
e65187cf-065f-476c-9e09-3698f78c229f
1162f2bf-6a89-4432-b598-7763378cd552
15543ed6-2057-4f1f-88f9-d97b5c31e2c1
936b1a57-c7de-4f42-b032-5d97bc541fe6
b7859aed-bb3c-4abe-8a98-444c194b260f

**Comment by Alex-NRCan** [comment]
Update from today. It seems to be related to projections. Still working on it...

**Comment by Alex-NRCan** [comment]
We have a fix inbound that will fix 13 out of the 24 mentioned.

The non-fixed ones (remaining ones are):

These ones seem to be a CORS issue that would need to be resolved likely server-side:
- eaf92ab8-c744-4f8f-a280-6987d4f6005e
- 6b1408b4-ea09-4a6a-b5dd-3db3493e1218
- 5a316fdc-3237-4ace-831e-67b4ca26a248
- 9eb51c7b-5336-4b50-a9f8-a6791bad15a9
- 01e01816-9f08-49eb-a5ac-7480195d90d4
- 41d008f7-f083-4801-b7c0-c2bf5a83a757
- 77f5bbd2-4f24-466f-b309-5b73659d32a3

These ones need further investigation. TIP: seems metadata doesn't support dynamic service? metadata extent is wrong?
- 94e38a40-085a-4c8a-a7c4-5b46e03702c3
- 8ce8abe1-d644-4134-baa9-389eebd63a05
- 522b07b9-78e2-4819-b736-ad9208eb1067
- 9e1507cd-f25c-4c64-995b-6563bf9d65bd

In summary then, these 4 are the only ones left to explain at the moment.

**Comment by Alex-NRCan** [comment]
Fixed the last 4. Only the CORS couldn't be fixed.
A PR is on the way.
This can be retested on next release.
